system/netdata.logrotate.in

```diff
@@ -0,0 +1,15 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		if service netdata status > /dev/null ; then \
+			service netdata restart > /dev/null; \
+		fi;
```

Comment:
Indirectly yes: when the daemon spawns a child, it leaves `/var/log/netdata/error.log` attached to the process `stderr`. So, let's say we have a bash plugin. This command:

```sh
echo >&2 "something failed"
```

will send this message to `/var/log/netdata/error.log`.
But the child is not aware of this. And the daemon is not in the middle anymore (as far as error.log is concerned).

So, we can do somehting in the daemon (SIGUSR1 maybe), but still `error.log` will be a problem.